src/controllers/userController.ts

```diff
@@ -7,11 +7,15 @@ import { v4 as uuidv4 } from 'uuid';
 import * as bcrypt from 'bcrypt';
 import * as email from '../middleware/email';
 import {EmailConfirmToken} from '../entity/EmailConfirmToken';
+import axios, {Method} from 'axios';
 
 const HOSTNAME = process.env.EXTERNAL_HOSTNAME;
 const CONFIRM_LINK = process.env.CONFIRM_LINK || '';
 const CONFIRM_TEMPLATE = process.env.CONFIRM_TEMPLATE || '';
 
+const DISCORD_URL = process.env.DISCORD_URL;
+const DISCORD_ROLES = process.env.DISCORD_ROLES;
```

Comment:
The naming of this variable is weird. We're only setting a _single_ role, right? Unless this is supposed to be a list that gets passed in somehow?